lib/tasks/load_csv.rake

```diff
@@ -0,0 +1,128 @@
+
+desc "destroy and reload all seed data"
+task :load_csv do
+  Rake::Task["initialize"].invoke
+  Rake::Task["load_customers"].invoke
+  Rake::Task["load_invoices"].invoke
+  Rake::Task["load_merchants"].invoke
+  Rake::Task["load_items"].invoke
+  Rake::Task["load_invoice_items"].invoke
+  Rake::Task["load_transactions"].invoke
+end
+
+task initialize: :environment do
```

Comment:
Do we need the require files within each task if you're requiring them here in the `initialize task`?

Reply:
@suzkiee Yeah we could remove them, but then we wouldn't be able to load one csv by itself. I don't really see us only loading one csv at a time though, so we could refactor later